test-cases/longevity/longevity-in-memory-36GB-1day.yaml

```diff
@@ -4,7 +4,7 @@ prepare_verify_cmd: ["cassandra-stress read                          cl=QUORUM n
 stress_cmd:         ["cassandra-stress mixed 'ratio(write=1,read=8)' cl=QUORUM duration=1400m -schema 'replication(factor=3)' -mode cql3 native -rate threads=200 -pop seq=1..21000000 -col 'size=FIXED(200) n=FIXED(5)' -log interval=10"]
 stress_read_cmd:    ["cassandra-stress read                          cl=QUORUM duration=1400m -schema 'replication(factor=3)' -mode cql3 native -rate threads=100 -pop seq=1..21000000 -col 'size=FIXED(200) n=FIXED(5)' -log interval=10"]
 
-run_fullscan: '{"ks_cf": "random", "interval": 30}' # 'ks.cf|random, interval(min)'
+run_fullscan: '{"mode": "select", "ks_cf": "random", "interval": 30}' # 'ks.cf|random, interval(min)'
```

Comment:
if the alternative operation is named `partition`, then it might be better having this parameter named `table` and not `select`

Reply:
and here

Reply:
Agreed - it's better to call it `table` as @yarongilor suggested. I changed the naming in all cases in the yamls.